src/api/mod.rs

```diff
@@ -134,14 +122,30 @@ impl CliCommands {
                     ApiCommand::ProductsV2(cmd) => cmd.run(global_options).await?,
                     ApiCommand::Releases(cmd) => cmd.run(global_options).await?,
                     ApiCommand::SigningKeys(cmd) => cmd.run(global_options).await?,
-                    ApiCommand::Users(cmd) => cmd.run(global_options).await?,
                     ApiCommand::Webhooks(cmd) => cmd.run(global_options).await?,
                 }
             }
+            CliCommands::Users(cmd) => cmd.run(global_options).await?,
             CliCommands::Upgrade(cmd) => cmd.run().await?,
             CliCommands::Config(cmd) => cmd.run(global_options).await?,
         };
 
         Ok(())
     }
+
+    pub(crate) fn print_missing_arguments(missing_arguments: Vec<String>) {
```

Comment:
I'm guessing this syntax makes it a pub fn but only within the context of the crate, yeah?

Reply:
Yep!  It's a sort form of the longer fully-qualified syntax of `pub(in crate)`.  The other short forms are `pub(self)` and `pub(super)`.  The visibility modifiers are covered [here] in The Rust Reference.